Directory.Packages.props

```diff
@@ -25,17 +25,17 @@
     <PackageVersion Include="LazyCache" Version="2.4.0" />
     <PackageVersion Include="Mages" Version="2.0.1" />
     <PackageVersion Include="Markdig.Signed" Version="0.27.0" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.3" />
-    <PackageVersion Include="Microsoft.Data.Sqlite" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="8.0.0-preview.23525.2" />
```

Comment:
Using preview analyzers? I'm guessing we can use a released version now?

Reply:
Oh I see they updated it to release version now! I can fix that